skimage/filters/thresholding.py

```diff
@@ -434,11 +434,12 @@ def threshold_isodata(image, nbins=256, return_all=False):
     # last bin of csumh, which is zero by construction.
     # So no worries about division by zero in the following lines, except
     # for the last bin, but we can ignore that because no valid threshold
-    # can be in the top bin. So we just patch up csumh[-1] to not cause 0/0
-    # errors.
-    csumh[-1] = 1
```

Comment:
The issue stemmed from this line of code. You can't assign something to dask  I just optimized that line away all together.